index.html

```diff
@@ -237,43 +261,47 @@ <h2>
                 </li>
               </ul>
             </div>
+            <button type="button" class="resume">Get my resume</button><br />
           </div>
         </div>
-
-        <button type="button" class="resume">Get my resume</button><br />
-
-        <ul class="languages">
-          <li class="lang">
-            Languages
-            <i class="fa fa-angle-down fa-lg"></i>
-          </li>
-          <li class="language-type">
-            <img src="images/js.svg" alt="javaScript-logo" />
-            <p>JavaScript</p>
-          </li>
-          <li class="language-type">
-            <img src="images/html.svg" alt="html-logo" />
-            <p>HTML</p>
-          </li>
-          <li class="language-type">
-            <img src="images/css.svg" alt="css-logo" />
-            <p>CSS</p>
-          </li>
-          <li class="frame">
-            Frameworks
-            <i class="fa fa-angle-right fa-lg"></i>
-          </li>
-          <li class="skil">
-            Skills
-            <i class="fa fa-angle-right fa-lg"></i>
-          </li>
-        </ul>
+        <div class="dt-about-me-languages">
+          <ul class="languages">
+            <li class="lang">
+              Languages
+              <i class="fa fa-angle-down fa-lg"></i>
+            </li>
+            <li>
+              <ul class="dt-language-types">
+                <li class="language-type">
+                  <img src="images/js.svg" alt="javaScript-logo" />
+                  <p>JavaScript</p>
+                </li>
+                <li class="language-type">
+                  <img src="images/html.svg" alt="html-logo" />
+                  <p>HTML</p>
+                </li>
+                <li class="language-type">
+                  <img src="images/css.svg" alt="css-logo" />
+                  <p>CSS</p>
+                </li>
+              </ul>
+            </li>
+            <li class="frame">
+              Frameworks
+              <i class="fa fa-angle-right fa-lg"></i>
+            </li>
+            <li class="skil">
+              Skills
+              <i class="fa fa-angle-right fa-lg"></i>
+            </li>
+          </ul>
+        </div>
```

Comment:
> - **Your implementation:**
![a1](https://user-images.githubusercontent.com/70483835/154792381-ce689e0d-83d6-40ae-8449-daf4586d5349.png)
> - **Figma design**
![s2](https://user-images.githubusercontent.com/70483835/154792473-c4350c68-141f-4242-8a85-f110f90dcbef.png)

- [ ] Please remove the dot icon beside language types. :+1: 

- [ ] In the about myself section, the top-right corner is curved, Please fix it. :+1: 

- [ ] In this section, your content is aligned more on the left side. You can try `margin` to give space properly on the right and left sides. Please fix it. :writing_hand:

Reply:
[OPTIONAL]

- [ ] Kindly, if you have enough time before submitting again, please fix the top-right corner to be curved as pointed out by the previous reviewer.

Reply:
[OPTIONAL]

- [ ] Kindly, if you have enough time before submitting again, please fix the top-right corner to be curved as pointed out by the previous reviewer.